src/amo/components/CollectionManager/index.js

```diff
@@ -89,6 +93,14 @@ export class CollectionManagerBase extends React.Component<Props, State> {
       this.setState({
         addonAddedStatus: props.hasAddonBeenAdded ?
           ADDON_ADDED_STATUS_SUCCESS : null,
+      }, () => {
```

Comment:
We decided not to use the `setState` second argument in https://github.com/mozilla/addons-frontend/issues/3077.

Reply:
oh? ok.. let me try another way...